ci/build.yml

```diff
@@ -6,7 +6,7 @@ image_resource:
   type: docker-image
   source:
     repository: golang
-    tag: 1.17.8
+    tag: 1.19.1
```

Comment:
Can bump to 1.19.2 - includes security fixes and linker fix (issue running linters in OS Monterey 1.12.6 when upgrading golangci version (latest is 1.50.0)

So in Makefile under lint target update golangci version to use 1.50.0

Reply:
Done